### PR TITLE
Remember Medtronic pump serial numbers using `node-keytar`

### PR DESCRIPTION
**Pull request by pazaan**
Also:
Fix typo: DEVICE_UNKOWN -> DEVICE_UNKNOWN.
Change default state of `medtronicSerialNumberValue` to empty string to avoid ReactJS warnings.

@krystophv - I think you're the best candidate to review this 😄

### CODE REVIEW

**Comment by krystophv** [comment]
[[Electron Uploader] Encrypt and store locally the user's Medtronic serial number](https://trello.com/c/hYZjKBPK/166-electron-uploader-encrypt-and-store-locally-the-users-medtronic-serial-number)

File: app/package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
+    "keytar": "^4.0.3",
```
Comment (krystophv) [inline app/package.json]: We'll need to just lock down this dependency.

Comment (pazaan) [inline app/package.json]: Ah, no worries. This is added by `yarn`, when I `yarn add`ed. It's a shame that `yarn add` doesn't have a flag to pin it down exactly. See also https://github.com/yarnpkg/yarn/issues/1676

Comment (krystophv) [inline app/package.json]: `yarn add` does have a flag for that: https://yarnpkg.com/lang/en/docs/cli/add/#toc-yarn-add-exact-e

Comment (pazaan) [inline app/package.json]: Cool. I saw that flag, but I thought it was tied to the `@version` syntax :)

File: app/components/Upload.js [hunk 2]
```diff
@@ -69,6 +79,28 @@ export default class Upload extends Component {
     this.onBlockModeInputChange = this.onBlockModeInputChange.bind(this);
     this.onCareLinkInputChange = this.onCareLinkInputChange.bind(this);
     this.onMedtronicSerialNumberInputChange = this.onMedtronicSerialNumberInputChange.bind(this);
+    this.onMedtronicSerialNumberRememberChange = this.onMedtronicSerialNumberRememberChange.bind(this);
+
+    this.populateRememberedSerialNumber();
+  }
+
+  componentWillMount() {
```
Comment (pazaan) [inline app/components/Upload.js]: @krystophv - I added this `componentWillMount` section after your review. Are you happy with this?

Comment (krystophv) [inline app/components/Upload.js]: Yeah, this LGTM

**Pull request merged**